cms/djangoapps/contentstore/outlines_backfill.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
I plopped most everything into a single file for simplicity. LMK if you feel I should move some/all of this code.

Reply:
Will do.